css/style.css

```diff
@@ -16,4 +18,147 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+nav{
+    display:flex;
+    flex-direction:row;
+    justify-content: space-around;
+}
+ul {
+    list-style: none;
+}
+
+.navbar{
+    background-color: white;
+    position: fixed;
+    width: 100%;
+    height: 4.375rem;
+    z-index: 10;
+    justify-content: space-around;
+    /* display: inline-block; */
+    /* margin-left: 1.563rem; */
+    /* margin-bottom: -300px; */
+}
+.navigates{
+    /* float: right; */
+    color: #838994;
+    font-size: 1.04rem;
+    margin-top: 1.7rem;
+    display: inline-block;
+    margin-left: 1.563rem;
+    font-weight: 500;
+}
+.logo{
+    height: 4.375rem;
+    width: 26px;
+}
+.navigates:hover{
+    cursor: pointer;
+    color: #333333;
+}
+.section1{
+    background-image: url(https://github.com/CodeYourFuture/karma-clone/blob/99d5c3fe44175a5563c36cd7806add33f0c3c670/img/first-background.jpg?raw=true);
+    /* background-position: center; */
+    background-repeat: no-repeat;
+    /* background-size: 100%; */
+    width: 100%;
+    /* height: 45.938rem; */
+    overflow: auto;
+    /* margin-top: -10px; */
+}
+.advertise h3{
+    color: white;
+    text-align: center;
+    font-weight: 300;
+    margin-top: 1.25rem;
+    font-size: 1.563rem;
+}
+.advertise h1{
+    color: white;
+    text-align: center;
+    font-weight: 300;
+    margin-top: 17.5rem;
+    font-size: 3.125rem;
+}
+.advertise{
+    /* padding: 10; */
+    /* margin-top: -800%; */
+}
+.buttons{
+    color: white;
+    /* display:flex;
```

Comment:
It's normally a good idea to remove comments when you're sure they're not needed, it stops the code from getting messy!

Reply:
I will do it for my next works.
thank you for the time you spent on my web.